server/common/src/main/java/org/eclipse/lsp/cobol/common/mapping/ExtendedText.java

```diff
@@ -129,7 +129,7 @@ public void insert(int line, ExtendedTextLine newLine) {
   public void insert(Range copyStatementRange, ExtendedText copybook) {
     Location location = mapLocation(copyStatementRange);
     clear(copyStatementRange);
-    insert(copyStatementRange.getStart().getLine() + 1, copybook, location);
+    insert(copyStatementRange.getStart().getLine(), copybook, location);
```

Comment:
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/33f22136a335aab59096469c3913ee7dc9bc067e/server/engine/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/copybooks/CopybookPreprocessorService.java#L124